### PR TITLE
Create logic for screenshot capture in mesheryctl ref

### LINKED ISSUE #5776: [Docs] Create screenshot column in `mesheryctl` reference section
#### Current State
Currently the docs is void of screenshot section in mesheryctl reference

#### Desired State
Screenshot section should be added in the mesheryctl ref section of docs site

---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by alphaX86**
Signed-off-by: Aadhitya A <aadhitya864@gmail.com>

**Description**

This PR fixes #5776 

**Notes for Reviewers**
Needs some links to test

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
Task linked: [CU-2hrb38z Create a screenshot field in the mesheryctl commands docs page](https://app.clickup.com/t/2hrb38z)

File: mesheryctl/doc/doc.go [hunk 1]
```diff
@@ -157,6 +167,14 @@ func GenMarkdownCustom(cmd *cobra.Command, w io.Writer, linkHandler func(string)
 	if err := printOptions(buf, cmd, name); err != nil {
 		return err
 	}
+
+	// If command has screenshot present
+	if picLine != "" {
+		buf.WriteString("## Screenshots\n\n")
+		buf.WriteString(picLine)
+		buf.WriteString("\n\n")
```
Comment (leecalcote) [inline mesheryctl/doc/doc.go:175]: Looks like an opportunity to capture a caption.

Example: "Display the list of all available applications."

https://twitter.com/mesheryio/status/1542556666733899776

Comment (alphaX86) [reply mesheryctl/doc/doc.go:175]: Ok, another line to be added then in the logic 🙂

Comment (leecalcote) [reply mesheryctl/doc/doc.go:175]: 👨‍🎤

**Review by leecalcote — APPROVED** [review]
Looking good. Thank you for today's example and demo, @alphaX86

**Comment by alphaX86** [comment]
I have added the pictures which @debo19 gave today... Almost added them in the code and in assets folder now

**Comment by l5io** [comment]
🚀 Preview for commit 14ccd007db2f987a0df00a41aeab04c8947dbea2 at: https://62d052146a2f0d009dc4bb43--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
[app list](https://www.linkedin.com/feed/update/urn:li:activity:6948323485509136384)
[app onboard](https://www.linkedin.com/feed/update/urn:li:activity:6947966582719414272)
[app view](https://www.linkedin.com/feed/update/urn:li:activity:6949738678227189760)

**Comment by leecalcote** [comment]
I don't seem to see the screenshots in the deployment preview...

**Review by leecalcote — APPROVED** [review]
Maybe the screenshots will show up after merge.

**Pull request merged**